src/controllers/user.controller.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const { userService } = require('../services/user.service.js');
+
+const getAll = async(req, res) => {
+  const users = await userService.getAll();
+
+  res.send(users.map(userService.normalizeUser));
+};
+
+const getOne = async(req, res) => {
+  const { userId } = req.params;
+  const foundUser = await userService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+  res.send(userService.normalizeUser(foundUser));
+};
+
+const add = async(req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const newUser = await userService.create(name);
+
+  res.statusCode = 201;
+  res.send(userService.normalizeUser(newUser));
+};
+
+const remove = async(req, res) => {
```

Comment:
Consider using a more descriptive function name like 'removeUser' instead of 'remove'